build/azure-pipelines/darwin/sql-product-build-darwin.yml

```diff
@@ -136,14 +136,14 @@ steps:
     continueOnError: true
     condition: and(succeeded(), eq(variables['RUN_TESTS'], 'true'))
 
-  # - script: |
-  #     set -e
-  #     node ./node_modules/playwright/install.js
-  #     VSCODE_REMOTE_SERVER_PATH="$(agent.builddirectory)/azuredatastudio-reh-web-darwin" \
-  #     yarn smoketest --web --headless  --screenshots "$(build.artifactstagingdirectory)/smokeshots"
-  #   displayName: Run smoke tests (Browser)
-  #   continueOnError: true
-  #   condition: and(succeeded(), eq(variables['RUN_TESTS'], 'true'))
+  - script: |
+      set -e
+      node ./node_modules/playwright/install.js
+      VSCODE_REMOTE_SERVER_PATH="$(agent.builddirectory)/azuredatastudio-reh-web-darwin" \
+      yarn smoketest --web --headless  --screenshots "$(build.artifactstagingdirectory)/smokeshots"
```

Comment:
We should either make separate sub-folders for the different platforms or have the platform be a part of the screenshot name to avoid confusion if there are failures on multiple platforms.